server/src/main/java/org/elasticsearch/action/fieldcaps/TransportFieldCapabilitiesAction.java

```diff
@@ -129,40 +129,19 @@ public TransportFieldCapabilitiesAction(
 
     @Override
     protected void doExecute(Task task, FieldCapabilitiesRequest request, final ActionListener<FieldCapabilitiesResponse> listener) {
-        executeRequest(
-            task,
-            request,
-            (transportService, conn, fieldCapabilitiesRequest, responseHandler) -> transportService.sendRequest(
-                conn,
-                REMOTE_TYPE.name(),
-                fieldCapabilitiesRequest,
-                TransportRequestOptions.EMPTY,
-                responseHandler
-            ),
-            listener
-        );
+        executeRequest(task, request, listener);
```

Comment:
I made @dnhatn add this the last time we changed this. I have copy-and-paste blindness and I said "please don't copy and paste this file, please make hooks, even if it's more work." Because I can never see the changes buried in copy-and-pasted code. And he, being a good sport, did.

Now I'm making a change and copy and pasting. I tried making hooks and though "this is unreadable garbage". It would have been so much worse. Not sure if I was wrong, but I'm pretty sure the hooks are just too hard to read once you make a change like this.

Now! The big copy-and-pasted bit is `doExecuteForked` - I could make a `class DoExecuteForked` with places we could override. That feels like it might be worth it. I'm not sure! I didn't do it so folks could look here and have opinions.